src/vussr.config.default.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   template: resolveApp('public/index.html'),
   outputPath: resolveApp('dist'),
   assetsPath: resolveApp('dist/assets'),
```

Comment:
Should this line not be removed then? If you fear breaking changes bump the version accordingly.

Reply:
Nope. The `assetsPath ` is the directory path of you file system. The `assetUrlPath `is the browser url, where you can find the assets. We have already another ticket open to clarify the variable names more clearly. But this will happen after MS2

Reply:
I figured with [this line](https://github.com/spring-media/vussr/pull/41/files#diff-9d9a6cd82f41984872a66a3ab0d440c4R9) you would detect and decide if the assetsUrlPath is **either** a relative path or a url and the magically use a CDN logic internally or not. I'd find that pretty cool.

Reply:
Yeah but this has nothing to do with the `assetsPath`. The `assetsPath` is the public accessible directory in the file system. All files in this folder can be accessed from the browser. The `assetsUrlPath` is the Browser URL (relative or CDN) under this URL all files under the public accessible directory (`assetsPath`) can be viewed.

Reply:
@veitbjarsch hat mir die technischen Gründe hierfür erklärt, finde aber das Wording nicht gut. Würde dann eher sowas schreiben wie `assetsDir` und `assetsPublicPath` um deutlich zu machen was die bedeuten. Anyway, sind nur meine 2 cents, ich approve